.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug
+about: You encountered something that is not working the way it should
+title: ''
+labels: type/bug
+assignees: ''
+---
+
+<!-- Instructions: -->
+<!-- 1. Remove sections/details you do not complete -->
+<!-- 2. Add sections/details useful to you -->
+
+|                    |                      |
+| ------------------ | -------------------- |
+| repro              | [link](https://todo) |
+| `node` version     | `10 | 12`            |
```

Comment:
Not sure why but the rendered version of `10 | 12` is broken :(

Reply:
Anyway will simplify to `...`